# CurriculumVitae
Nuclear Safety Department, Nuclear Science Center of Tunisia.
2020 Ariana, Technopole Sidi Thabel - Tunis.

<div id="webaddress">
kmezhoud@gmail.com
| 
00216 25 90 99 01

</div>


## Currently

I'm a research assistant in Nuclear Science Center of Tunisia. My main goal is to assess the risk of Low radiation exposure of Radiobiologist workers in Hospitals, Airport or other work places that have ionizing Rays. The Synopsis of my work is to predict the OMICS frontiers of healthy and cancer patients using Multi-Assays of genomics data. I'm developing tools to explore, visualize and interpret data from public repositories as [BiocPortal](http://www.cbioportal.org/).

### Previous interest
1. Plasma proteome analysis of radio-biologist workers exposed to low ionizing radiation. I used Proteome depletion and nano LCMSMS to elucidate peculiar protein-protein interaction network involved on low radiation stress. This network is enriched by biological process and cancer gene expression profiles to predict specific biomarkers, wich main biological process and the most probable cancer for low radiation exposure. This [supplementary Material](https://wiki.ubuntu.com/kmezhoud/Radiobiology_risks) describes in detail my workflow of post-proteomics data analysis and enrichment.

2. Microcystin is a cyanotoxin that can be found in sea foods. It is hepato-toxic and can induce liver cancer. I worked on it to elucidate the molecular mechanisms to induce cancer using Proteomics techniques (*in vivo* Assays (Fish, Rodent)), 2D SDS-PAGE, LC-MSMS, bioinformatics).

## Education

`2013-`
<a href="https://www.coursera.org/">__elearning__</a> Bioinformatics, Biostatistics, System Biology, Biomedical system, and Integrative Biology.

`2013`
__Faculté des Sciences de Tunis__ Habilitation (HDR)

`2003-07`
__Museum National d'Histoire Naturelle (Paris, France)__ PhD Proteomics

`2002-03`
__Museum National d'Histoire Naturelle (Paris, France)__ DEA in Ecosystems and toxins interactions

`2001-02`
__National Veterinary  School of Alfort (Paris, France)__ Master in Total Quality and Management

`1995-00`
__National Agriculture Institute of Tunisia__ Engineer in Food Technologies




## Selected Publications

<!-- ### Journals -->

`2014`
__Karim Mezhoud__, Amina Sakly, Hassen Ben Cheikh, Mouldi Saïdi, and Marc Edery. Radiobiology Worker Risk Assessment Using Stress Indicators and Proteomics. *Int. J. Low Radiation* Vol.9, No.3, 2014.

`2013`
__Karim Mezhoud__ Graphical identification of cancer-associated gene sub-networks based on small proteomics data sets. *OMICS: A Journal for Integrative Biology*  2013 17(7): 393-397.

`2011`
__Karim Mezhoud__, Puiseux-Dao S., Edery M., Toxins and stress in fish: proteomic analysis and response network. 2011 Review, *Toxicon*, 57(7-8), 959-969.

`2009`
__Mezhoud K.__, Sghaier H. and Barkallah I. (2009) " Deciphering peculiar protein-protein interacting modules in *Deinococcus radiodurans* "*Biology Direct*, 4:12

`2009`
Malécot M., __Mezhoud K.__, Marie A., Praseuth D., Puiseux-Dao S. et Edery M. " Toxicoproteomics: a way to unravel response pathways  induced by a toxin" *Aquatic Toxicology*, (2009) 94(2):153-61. 



`2008`
__Mezhoud K.__, Praseuth D., François J.C., Bernard C., and Edery M. " Global quantitative analysis of protein expression and phosphorylation status in the liver of the medaka fish (*Oryzias latipes*) exposed to microcystin-LR I. Balneation study " *Aquatic Toxicology*, (2008) 86 : 166 - 175

`2008`
__Mezhoud K.__, Bauchet A.L., Praseuth D., Marie A., François J.C., Fontaine J.J., Jaeg J.P., Cravedi J.P., Puiseux-Dao S., Bernard C. and Edery M. " Proteomic and phosphoproteomic analysis of cellular responses  to microcystin-LR in medaka fish", *Oryzias latipes*. 2 -  Gavage  study" *Toxicon* 51 (2008) 1431 - 1439.

## R Packages
[__canceR__](http://www.bioconductor.org/packages/release/bioc/html/canceR.html) is a Graphical User Interface for accessing and modeling the Cancer Genomics Data of MSKCC.

[__bioCancer__](https://github.com/kmezhoud/bioCancer) is a Shiny App to explore interactively Multi-Assay of Cancer Genomic Data.


## Technical skills

R | Statistics | LaTeX | Github | GNU/Linux | Bash | regex |Proteomics | Mass Spectrometry | Biochemistry |. 


## Language
French | Arabic | English

## Reference

<!--
[Levi Waldron](http://waldronlab.org/) works at CUNY School of Public Health at Hunter College 2180 3rd Ave Room, 538 New York, NY 10035 levi.waldron@hunter.cuny.edu
-->


